config/routes.rb

```diff
@@ -19,12 +19,9 @@
   namespace :api do
     namespace :internal do
       api_version(module: "V1", path: { value: "v1" }, default: true) do
-        resources :prosecution_cases, only: [:index]
-        resources :prosecution_case_laa_references, path: "laa_references", only: %i[create]
         resources :defendants, only: %i[update show]
         resources :prosecution_case_representation_orders, path: "representation_orders", only: [:create]
         resources :court_application_representation_orders, only: [:create]
```

Comment:
I've removed `court_application_representation_orders` as well, in a different PR, anyway this was open before than mine.
I believe you can also remove the fixture file `spec/fixtures/files/court_application_search_result.json`

Reply:
I've been through every deleted `file_fixture` call to check which fixtures are no longer referenced.